.github/workflows/generate-models.yml

```diff
@@ -21,6 +15,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Checkout model repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'amzn/selling-partner-api-models'
+          path: 'tmp/selling-partner-api-models'
```

Comment:
Any reason for using local `tmp` rather than global `tmp`?

If we store it in local `tmp` then we need to make sure it is also in `.gitignore`.

Reply:
> global tmp

You mean `/tmp`? I tried but GH actions didn't might allow us access outside `/home/runner/work/selling-partner-api-sdk/selling-partner-api-sdk` ([actions log](https://github.com/nguyentoanit/selling-partner-api-sdk/runs/2516389100?check_suite_focus=true#step:3:12)). 

> If we store it in local tmp then we need to make sure it is also in .gitignore.

Sure. I verified [them](https://github.com/ScaleLeap/selling-partner-api-sdk/blob/master/.gitignore#L108) before.

Reply:
Ah, okay, great. I understand. They've locked up the system. No problem!